Dockerfile

```diff
@@ -1,16 +1,59 @@
+# Support setting various labels on the final image
+ARG COMMIT=""
+ARG VERSION=""
+ARG BUILDNUM=""
+
 # Build Geth in a stock Go builder container
-FROM golang:1.16-alpine as builder
+FROM golang:1.17-alpine as builder
 
-RUN apk add --no-cache make gcc musl-dev linux-headers git bash
+RUN apk add --no-cache make gcc musl-dev libc6-compat linux-headers git bash
 
 ADD . /go-ethereum
-RUN cd /go-ethereum && make geth
+RUN cd /go-ethereum && go run build/ci.go install -static ./cmd/geth
```

Comment:
Why not add another target in Makefile?

Reply:
The docker image uses alpine, other linux do not install the musl static library, the static link is best only in the docker environment

Reply:
now Makefile add new `all-static` and `geth-static` build target

Reply:
It make sense, but we have to test them in different architectures.